spec/models/reports/boxi/orders_serializer_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Reports
+  module Boxi
+    RSpec.describe OrdersSerializer do
+      let(:dir) { "/tmp/test" }
+      let(:csv) { double(:csv) }
+      let(:headers) do
+        %w[
+          RegistrationUID
+          OrderUID
+          OrderCode
+          PaymentMethod
+          TotalCharge
+          Description
+          MerchantID
+          CreationTimestamp
+          LastModifiedTimestamp
+          LastModifiedBy
+        ]
+      end
+      subject { described_class.new(dir) }
+
+      before do
+        expect(CSV).to receive(:open).and_return(csv)
+        expect(csv).to receive(:<<).with(headers)
+      end
+
+      describe "#add_entries_for" do
+        let(:registration) { double(:registration) }
+
+        it "creates an entry in the csv for each order_item in the registration" do
```

Comment:
I'm assuming this should be 'order'!

```suggestion
        it "creates an entry in the csv for each order in the registration" do
```

Reply:
Oh no again XD